.github/workflows/publish_patch.yaml

```diff
@@ -0,0 +1,29 @@
+name: Publish Patch
+
+# Run 15 minutes after the hour, every hour.
+# We use this instead of the top of the hour to avoid heavy-traffic times on github.
+on:
+  schedule:
+    - cron: '*/15 */1 * * *'
+
+jobs:
+  publish_patch:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Choose Clock Face
+      - run: |
+        # Choose a clock face based on the current hour
+        # Even hours = particle
+        # Odd hours = generative
+
+        current_hour=$(date +%H)
+        if [ $(($current_hour % 2)) -eq "0" ]; then
+          clock_face=particle
```

Comment:
Does this work? I thought you had to explicitly pass the variable to `GITHUB_ENV` because steps are run in their own separate process.

Reply:
Quite possibly not – I just wanted to get this up as a draft 1) for tips like this and 2) because I'm about to sign off for the day.